cmd/config-migrator/main.go

```diff
@@ -68,6 +68,18 @@ func generateMigratedConfigs(input config.DataWithInfo) []config.DataWithInfo {
 		return nil
 	}
 
+	var newBaseRPMImages map[string]api.ImageStreamTagReference
+	for k, baseRPMImage := range futureConfig.BaseRPMImages {
+		if newBaseRPMImages == nil {
```

Comment:
Just initialize it in L17_

Reply:
L71 right?

I did not do it there because i wanna tell between nil vs emptyMap. Only nil will make the yaml serialization work for `omitempty`? Let me test it.

Reply:
You are right.
EmpytMap works too.

PR updated.